src/main/java/uk/nhs/prm/e2etests/model/response/PdsAdaptorResponse.java

```diff
@@ -1,11 +1,30 @@
 package uk.nhs.prm.e2etests.model.response;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 
-@Data
+import java.util.Objects;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
 public class PdsAdaptorResponse {
     private Boolean isSuspended;
     private String currentOdsCode;
     private String managingOrganisation;
     private String recordETag;
-}
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
This can be replaced by Lombok's default Equals implementation as it is comparing every field in the class.